lib/auth/auth_with_roles.go

```diff
@@ -62,6 +62,46 @@ func (a *ServerWithRoles) ValidateIdemeumServiceToken(ctx context.Context, Servi
 	return a.authServer.ValidateIdemeumServiceToken(ctx, ServiceToken, TenantUrl)
 }
 
+func (a *ServerWithRoles) GetUserSessions(ctx context.Context, UserId string, DeviceId string, TokenId string) ([]types.WebSession, error) {
+	log.Infof("Getting all the user's sessions for user %v", UserId)
+	allWebSessions, err := a.WebSessions().List(ctx)
```

Comment:
This implementation should be sufficient for now. When we have tenants with a lot of users it might now be. This is async call anyhow and each tenant will use its own DynamoDb table, so should not be that terrible even with a lot of users. Entries are expired and cleaned up.

Reply:
If it was app management code, i would be much more comfortable for approach that loads all database rows for a tenant.
In appmanagement, we have much better level of monitoring, experience, skillset, ability to turnaround time so that we can identify customer issue and rollout quickly. Same is not true for teleport.

@go-idemeum I dont want slow your progress, I approve hesitatingly.